### DISCUSSION TITLE
I hope the government will give an optimization of Dual CPU and GPU. Is the performance in this aspect not ideal at present

### DISCUSSION THREAD

**Discussion by s1mple1122** [filecoin-project/lotus#6043] [Ideas] [answered]
I tested the relevant performance, it is not ideal, at present I have nothing to do

**Comment by dd45e640b42e6da7da96faee3996ef7c** [comment, marked as answer]
for dual GPUs consider using them side by side with 2 or more workers with each one GPU. 

not sure what you mean with dual CPUs, the workers use them quite fine. If you are referring to the PC1 processing step: it will only be able to use a single CPUs cores. And even on single CPU it is limited to a single core complex
